libs/auth-api-lib/seeders/data/scope-regulations-admin.ts

```diff
@@ -23,8 +23,8 @@ export const up = compose(
     accessControlled: true,
     delegation: {
       custom: false,
-      legalGuardians: false,
       procuringHolders: false,
+      legalGuardians: false,
```

Comment:
This one also should have custom true and procuringHolders true.

Reply:
ah alright 👀